addons/binding/org.openhab.binding.pentair/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Missing about.html, see http://docs.openhab.org/developers/development/guidelines.html

And see the red symbol meaning that it misses a newline